lib/src/source_visitor.dart

```diff
@@ -76,14 +74,42 @@ class SourceVisitor extends ThrowingAstVisitor {
     return target is SimpleIdentifier && _looksLikeClassName(target.name);
   }
 
+  /// Whether [name] appears to be a type name.
+  ///
+  /// Type names begin with a capital letter and contain at least one lowercase
```

Comment:
[out of scope] Is it sensible to consider `_A` or `A` to be a class name?

Reply:
Honestly, I don't think it matters much one way or the other. The only way this could come into play is a (1) static method call that (2) didn't fit on one line on (3) an identifier whose name (4) is a single capital letter.